traffic_ops/traffic_ops_golang/cfgdiffs.go

```diff
@@ -0,0 +1,303 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/apache/incubator-trafficcontrol/lib/go-log"
+	"github.com/apache/incubator-trafficcontrol/traffic_ops/traffic_ops_golang/auth"
+
+	"github.com/jmoiron/sqlx"
+)
+
+const CfgDiffsPrivLevel = auth.PrivLevelReadOnly
+const CfgDiffsWritePrivLevel = auth.PrivLevelOperations
+
```

Comment:
these PrivLevel consts should go away -- we're using the `auth.PrivLevel...` consts to control individual routes in `routes.go`